tests/rust/wasm32-wasip3/wit/deps/wasi-cli-0.3.0-rc-2025-08-15/package.wit

```diff
@@ -0,0 +1,202 @@
+package wasi:cli@0.3.0-rc-2025-08-15;
```

Comment:
I'm not a wit expert, but can we, instead of copying the files, pull them as a submodule from the original repo: https://github.com/WebAssembly/wasi-cli/ ?